pom.xml

```diff
@@ -43,6 +43,11 @@
             <version>2.2.1</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
Should the scope for this dependency be `provided`?

Reply:
No, these are not accessible on the classpath, so provided does not work. Tested it out, just in case, it fails with provided.